### PR TITLE
Misc. updated re: CakePHP 3

### PR DESCRIPTION
**Pull request by jadb**
Thanks for merging this PR.

/cc @markstory @lorenzo

### CODE REVIEW

**Comment by lorenzo** [comment]
Looks good :+1:

**Comment by jadb** [comment]
@ziadoz anything I can do to speed up the merge of this PR?

**Comment by ziadoz** [comment]
Sorry. I've been really busy lately. I do plan to start looking through and merging PRs again next month though, once things have calmed down.

**Comment by dereuromark** [comment]
:+1:

**Comment by DSchalla** [comment]
:+1: - 10/10, would use again.

**Comment by jadb** [comment]
ping @ziadoz

**Comment by lorenzo** [comment]
@ziadoz Anything I could help with?

**Comment by davidyell** [comment]
Great idea! :+1:

**Comment by mamchenkov** [comment]
:+1:

**Comment by jadb** [comment]
ping!!

**Comment by dereuromark** [comment]
Whats going on here?

**Comment by hmic** [comment]
:+1:

**Comment by lorenzo** [comment]
Pretty please? :dancers:

**Comment by jadb** [comment]
@ziadoz - remember me? still waiting... https://twitter.com/jadb/status/601422773597044737

File: README.md [hunk 1]
```diff
@@ -113,13 +113,15 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
...
-
+* [CakePHP CRUD](https://github.com/friendsofcake/crud) - A Rapid Application Development (RAD) plugin for CakePHP.
+* 
 ## Components
 *Standalone component from web development frameworks and development groups.*
 
 * [Symfony2 Components](http://symfony.com/doc/master/components/index.html) - The components that make Symfony2.
 * [Zend Framework 2 Components](https://packages.zendframework.com/) - The components that make ZF2.
 * [Aura Components](http://auraphp.github.com/) - A package of PHP 5.4 components.
+* [CakePHP Plugins](http://plugins.cakephp.org/) - A directory of CakePHP plugins.
```
Comment (ziadoz) [inline README.md]: I feel like this probably belongs under the `Framework Extras` group. This category is meant to link to the main documentation for the components included in each framework (or working group). You could link to the main CakePHP manual in this section though (like Symfony 2) if you like.

File: README.md [hunk 2]
```diff
@@ -393,6 +395,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Hoa EventSource](https://github.com/hoaproject/Eventsource) - An event source library.
 * [Evenement](https://github.com/igorw/evenement) - An event dispatcher library.
 * [Event](https://github.com/thephpleague/event) - An event library with a focus on domain events.
+* [Event](https://github.com/cakephp/event) - An event dispatcher library (part of CakePHP).
```
Comment (ziadoz) [inline README.md]: Could you pick an acronym for CakePHP, something short (see the other frameworks for examples), and put in brackets after CakePHP (e.g. `(CP)`) in frameworks, and then replace all the `(part of CakePHP)` bits with it too. Thanks.

**Comment by ziadoz** [comment]
@jadb Apologies for taking so long to reply. If you could get the branch up-to-date and make the minor amends I've suggested then I'll get this merged. Thanks.

File: README.md (hunk 2 above)
Comment (jadb) [inline README.md]: Sure thing. Will get you those today!

**Comment by davidyell** [comment]
:tada:

**Comment by jadb** [comment]
@ziadoz  - fixed most of the stuff as per your suggestion. Included `cake` in some of the package names like zend/doctrine have done (ie `Zend Cache`) and `(CP)` like `(ZF)`.

I kept the plugins under the components section as when I checked the first one on the list, (ZF2), it links to a list of Zend components, just like the one for CakePHP links to plugins for CakePHP. The documentation/book is not really a list of components like the rest of links are showing so maybe I should add the book elsewhere? 

In any case, thanks a lot for picking that old PR back up! Looking forward to getting those on the list :)

**Comment by ziadoz** [comment]
Thanks, I've merged this.

**Comment by jadb** [comment]
Thanks @ziadoz - have you re-considered getting some help to maintain this awesome list? I know we had mutually mentioned it but I guess you got distracted/busy ;)

**Comment by ziadoz** [comment]
@jadb I started writing some collaborator guidelines shortly after you contacted me through Twitter. I need to finish them up and get some feedback on them. Once I've done that I'll bring people onboard who are interested in helping out.

**Comment by jadb** [comment]
Sweet @ziadoz! Keep me posted, am in ;)

**Pull request merged**